### ISSUE TITLE
Add option to remove volumes on stop

### ISSUE THREAD

**Issue by markmsmith** [testcontainers/testcontainers-node#29]
### Feature Request
When running an official image (such as postgres) you're stuck with the `VOLUME`s that it declares. This has the unfortunate effect of having your tests accumulate anonymous volumes, as can be seen with `docker volume ls`.  It would be really helpful if there was an option to remove any associated volumes when a container is stopped (which would actually be handled by passing the option to the underlying call to dockerode's `remove()`):
https://docs.docker.com/engine/api/v1.37/#operation/ContainerDelete

#### Workaround:
It's pretty nasty, but you can set the underlying container's default options for remove (but obviously it would be better not to break the abstraction):
```ts
(pgContainer as any).container.container.defaultOptions.remove.v = true;
```

**Comment by cristianrgreco** [comment]
Available in 1.3.0.

**Comment by markmsmith** [comment]
Confirmed working in 1.3.1,
```ts
await testContainer.stop();
```
will now remove any volumes associated with container (as defined in its dockerfile) by default.  
This behavior can be overridden by passing a config object to `stop`, eg:
```ts
await testContainer.stop({
  removeVolumes: false
});
```
in the case that you want to look at it for debugging purposes.
